yo1k/tic_tac_toe/core/util.py

```diff
@@ -1,8 +1,6 @@
 def eq(cls):
     """ Class decorator providing generic comparison functionality """
```

Comment:
Spaces after and before `"""` are not needed, and you don't have them in other places. Also, there is not `.` at the end of the sentence. When you fix this two issues, please feel free to just go ahead and merge.

Reply:
Done.